libexec/setzer/setzer-price-avaxusd

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+pair=avaxusd
+sources=(
+  binance
+  okex
+  huobi
+  gateio
+  bitfinex
+)
+
+case $1 in
+  bitfinex) {
+    setzer x-price "$1" avax%3A:usd
```

Comment:
why is it `avax%3A:usd` ?

Reply:
Workaround for the ":" parsing for a 4 letter symbol. Bitfinex requires a ":" in the URL, so I encode it.
`https://api.bitfinex.com/v2/ticker/tAVAX:USD`